### ISSUE TITLE
Wealth effect <-> Payment size effect in case D

### ISSUE THREAD

**Issue by bermanjoe** [LMLhub/Discounting#10]
In case D we can add that the preference reversal depending on initial wealth can be viewed alternatively as a payment size effect (which is documented in the literature, sometimes under the name salient payoffs).

**Comment by diomavro** [comment]
Got a reference?

**Comment by alex-adamou** [comment]
Yes, Green & Myerson (1996) discusses amount-dependent discounting on pages
499-500. They give some references.

This guy <https://twitter.com/inferencelab/status/1183330857135824898>
might know something (although I don't understand his sketch). He calls it
the "magnitude effect" and that phrase certainly generates hits when
searched, e.g. about pigeons <https://www.ncbi.nlm.nih.gov/pubmed/22229590>.
Dio, would you like you ask him?

Also, we should make sure we understand our own prediction. In case D,
people switch from preferring the earlier to the later payment, i.e. they
discount less steeply, as they get wealthier. Another way of making people
"wealther" in this specification is to scale both payments down by the same
factor (between 0 and 1) while leaving wealth unchanged. So it would seem
that case D predicts people discount less steeply as payment size
decreases. If so, this is the opposite of the behaviour identified in the
literature. Please could someone check I have this the right way around?

Alex
--
Alex Adamou | Director
London Mathematical Laboratory
+447779324026

**Comment by bermanjoe** [comment]
Our model indeed predicts (in case D) an opposite magnitude effect: as the later payment gets larger, the discount rate increases. Green and Myerson find the opposite in several studies (This is the clearest one: https://link.springer.com/article/10.3758/BF03211314). They ask people (no real incentives, just questions) to price later payments relative to immediate ones, and impute their hyperbolic and exponential discount rates accordingly. They find that the rates stop decreasing when the later, larger payment reaches $25K (in the 1990s).

There are two main issues with this kind of evidence with respect to our paper:
1. It is not an experiment, so in reality things might be different
2. The (hypothetical) earlier payment is immediate, a case in which our model breaks in any case
3. This is not really a choice problem, but a pricing problem - the subjects are asked to specify the equivalent immediate payment to the later, larger payment. It is different contextually from a choice problem.

In the literature review of Frederick et al. (Time Discounting and Time Preference: A Critical Review), they write the following:
> The “Magnitude Effect” (small outcomes are discounted more than large ones):
Most studies that vary outcome size have found that large outcomes are discounted at a lower rate than small ones (Ainslie and Varda Haendel 1983; Benzion, Rapoport, and Yagil 1989; Green, Fristoe, and Myerson 1994; Green, Astrid Fry, and Myerson 1994; Holcomb and Nelson 1992; Kirby 1997; Kirby and Marakovic 1995; Kirby, Nancy Petry and Warren Bickel 1999; Loewenstein 1987; Raineri and Rachlin 1993; Marjorie K. Shelley 1993; Thaler 1981). In Thaler’s (1981) study, for example, respondents were, on average, indifferent between $15 immediately and $60 in a year, $250 immediately and $350 in a year, and $3000 immediately and $4000 in a year, implying discount rates of 139 percent, 34 percent, and 29 percent, respectively.

The question is whether we would like to address this point in the paper or not. Probably not.

**Comment by ole-peters** [comment]
It sounds to me like a case of using one concept and dropping a bunch of
observations into it -- appropriately or otherwise.

Scales do exist in prices for humans. Small payments may well mean payments
that are small compared to disposable income, something additive. Large
payments (large compared to disposable income) are relevant for savings and
investments, which are largely multiplicative.

This is most likely confusion arising from using concepts -- multiplicative
discount rates -- outside the context where they apply (investment
decisions). Probably a can of worms we don't want to open in this paper.

If someone wants to unpuzzle the magnitude effect, that's another study.

O

PI Ergodicity Economics | London Mathematical Laboratory
External Professor | Santa Fe Institute
@ole_b_peters <https://twitter.com/ole_b_peters>

ergodicityeconomics.com
